### ISSUE TITLE
Option followLocation not working

### ISSUE THREAD

**Issue by CedricYii** [yiisoft/yii2-httpclient#20]
I think the option followLocation is not working, here is how i try to use it:

```php

        $httpclient = new Client(['baseUrl' => 'http://localhost']);

        $response = $httpclient->createRequest()
            ->setUrl('login.php')
            ->setData(['login' => 'user', 'passwd' => 'xyz'])
            ->setMethod('post')
            ->setOptions([
                          'followLocation'  => true,
                          //'maxRedirects'    => 2,
                      ])
            ->send();

```

On success, login.php give a new Location url in header but $response does not follow it.

**Comment by klimov-paul** [comment]
Resolved by commit b983312b60ec9251eb9abca76fa89014f8828388

**Comment by scryba** [comment]
Doesn't seem to work in this current version.
